install.sh

```diff
@@ -121,9 +123,27 @@ elif lspci | grep -E -i '(vga|3d|display)' | grep -i amd > /dev/null; then
 			vulkan-radeon \
 			lib32-vulkan-radeon \
 			xf86-video-amdgpu
+	
+	elif lspci -nnk | grep -i vga -A3 | grep 'Kernel modules: amdgpu' > /dev/null; then
+		echo "AMD graphics card with amdgpu kernel module detected, installing drivers..."
+		pacman --noconfirm -S \
+			mesa \
+			lib32-mesa \
```

Comment:
mesa and lib32-mesa are not needed here

Reply:
For Proton are required, I will re tweak base packages